Add docs on solving exercises and feedback (#181)
exercism/docs: using/README.md

# Using Exercism

> These documents help learn how to make the most of your experience on Exercism.

Reviewer: ```suggestion
These documents describe how to make the most of your experience on Exercism.
```

Or e.g.:
> These documents try to teach you how [...]

> These documents help you learn how [...]

> - [Getting Feedback](./feedback/README.md)

Reviewer: ```suggestion
- [Getting Feedback](./feedback/README)
```

> - [Solving Exercises](./solving-exercises/README.md)

Reviewer: ```suggestion
- [Solving Exercises](./solving-exercises/README)
```
